src/dht/put.js

```diff
@@ -16,8 +16,12 @@ module.exports = configure(({ ky }) => {
     const searchParams = new URLSearchParams(options.searchParams)
     if (options.verbose != null) searchParams.set('verbose', options.verbose)
 
-    key = Buffer.isBuffer(key) ? encodeBufferURIComponent(key) : encodeURIComponent(key)
-    value = Buffer.isBuffer(value) ? encodeBufferURIComponent(value) : encodeURIComponent(value)
+    key = Buffer.isBuffer(key)
```

Comment:
Similar to the curly brackets that have been added, these just look like whitespace changes - please revert.

Reply:
Will do!